reference/5.1/Microsoft.PowerShell.Core/About/about_PSSnapins.md

```diff
@@ -14,29 +14,27 @@ Describes  Windows PowerShell snap-ins and shows how to use and manage them.
 
 ## Long description
 
-A Windows PowerShell snap-in is a Microsoft .NET Framework assembly that
-contains Windows PowerShell providers and\/or cmdlets. Windows PowerShell
-includes a set of basic snap-ins, but you can extend the power and value of
-Windows PowerShell by adding snap-ins that contain providers and cmdlets that
-you create or get from others.
+A Windows PowerShell snap-in is a Microsoft .NET Framework assembly that can
+contains Windows PowerShell providers and cmdlets. Windows PowerShell includes
+a set of basic snap-ins, but you can extend the power and value of Windows
+PowerShell by adding snap-ins that contain providers and cmdlets that you
+create or get from others.
 
 When you add a snap-in, the cmdlets and providers that it contains are
 immediately available for use in the current session, but the change affects
 only the current session.
 
 To add the snap-in to all future sessions, save it in your Windows PowerShell
-profile. You can also use the Export-Console cmdlet to save the snap-in names
-to a console file and then use it in future sessions. You can even save
-multiple console files, each with a different set of snap-ins.
+profile. You can also use the `Export-Console` cmdlet to save the snap-in names
+to a console file and then use that saved configuration in future sessions.
```

Comment:
With the only remaining snap-in being Core, which is always loaded, is this information still relevant/useful? Or are we just keeping it for consistency, or are there still snap-ins we need to keep this information for?

Reply:
None we care about.